src/main/java/mil/dds/anet/resources/PositionResource.java

```diff
@@ -64,7 +64,9 @@ private void assertCanUpdatePosition(Person user, Position pos) {
     if (pos.getType() == PositionType.ADMINISTRATOR) {
       AuthUtils.assertAdministrator(user);
     }
-    AuthUtils.assertSuperUserForOrg(user, pos.getOrganizationUuid(), true);
+    if (pos.getOrganizationUuid() != null) {
```

Comment:
Doen't this change the server-side behavior?

Reply:
Yes, scenario:
An admin tries to create a position, somehow client-side allows the position data which doesn't have organization field:
- Before PR behavior: Server sends "You do not have permissions to do this". ( Due to org.uuid being null, and super user checks failed)
-  After PR behavior: Server sends "An position must belong to an organization".

This piece of code is unreachable in current candidate:

```
    if (pos.getOrganizationUuid() == null) {
      throw new WebApplicationException("A Position must belong to an organization",
          Status.BAD_REQUEST);
    }
```

Reply:
This change is a tad dangerous in my opinion; I would prefer something like this in `assertCanUpdatePosition`:
```
    if (pos.getOrganizationUuid() == null) {
      throw new WebApplicationException("A Position must belong to an organization",
          Status.BAD_REQUEST);
    }
    AuthUtils.assertSuperUserForOrg(user, pos.getOrganizationUuid(), true);
````
to keep all assertions in one place. Otherwise you run the risk of someone changing `validatePosition` and suddenly super-users might be able to create or update positions wihout an organization.

Reply:
Yes, i was curious about how the order and grouping should be. Do you copy-paste the `null` check or cut-paste  from `validatePositions`?

Reply:
One could argue that `pos.getOrganizationUuid()` not being `null` is an assertion, so simply move that `if` from `validatePosition` to `assertCanUpdatePosition`.